test/scenarios/fork/ynETHWithdrawals.t.sol

```diff
@@ -0,0 +1,544 @@
+        assertEq(ynETHRedemptionAssetsVaultInstance.hasRole(ynETHRedemptionAssetsVaultInstance.REDEEMER_ROLE(), _redeemer), true, "testInitializationVault: E2");
+        assertEq(ynETHRedemptionAssetsVaultInstance.hasRole(ynETHRedemptionAssetsVaultInstance.PAUSER_ROLE(), _admin), true, "testInitializationVault: E3");
+        assertEq(ynETHRedemptionAssetsVaultInstance.hasRole(ynETHRedemptionAssetsVaultInstance.UNPAUSER_ROLE(), _admin), true, "testInitializationVault: E4");
+        assertEq(ynETHRedemptionAssetsVaultInstance.paused(), false, "testInitializationVault: E5");
+    }
+
+    function testRedemptionRate() public {
+        if (!isHolesky) return;
+
+        uint256 _expected = yneth.previewRedeem(1 ether);
+        assertEq(ynETHRedemptionAssetsVaultInstance.redemptionRate(), _expected, "testRedemptionRate: E0");
+    }
+
+    function testAvailableRedemptionAssets(address _redeemer, uint256 _dealAmount) public {
+        if (!isHolesky) return;
+
+        uint256 _expected = address(ynETHRedemptionAssetsVaultInstance).balance;
+        assertEq(ynETHRedemptionAssetsVaultInstance.availableRedemptionAssets(_redeemer), _expected, "testAvailableRedemptionAssets: E0");
+
+        vm.deal(address(ynETHRedemptionAssetsVaultInstance), _dealAmount);
+        assertEq(ynETHRedemptionAssetsVaultInstance.availableRedemptionAssets(_redeemer), _expected + _dealAmount, "testAvailableRedemptionAssets: E1");
+    }
+
+    function testTransferRedemptionAssets(address _to, uint256 _amount, uint256 _dealAmount) public {
+        if (!isHolesky) return;
+
+        vm.deal(address(ynETHRedemptionAssetsVaultInstance), _dealAmount);
+        uint256 _balanceBefore = address(ynETHRedemptionAssetsVaultInstance).balance;
+        uint256 _toBalanceBefore = address(_to).balance;
+
+        if (_balanceBefore < _amount) {
+            vm.expectRevert(abi.encodeWithSelector(InsufficientAssetBalance.selector, ETH_ASSET, _amount, _balanceBefore));
+            vm.prank(address(ynETHWithdrawalQueueManager));
+            ynETHRedemptionAssetsVaultInstance.transferRedemptionAssets(_to, _amount);
+        } else {
+            vm.expectRevert(abi.encodeWithSelector(AccessControlUnauthorizedAccount.selector, address(this), ynETHRedemptionAssetsVaultInstance.REDEEMER_ROLE()));
+            ynETHRedemptionAssetsVaultInstance.transferRedemptionAssets(_to, _amount);
+
+            vm.prank(address(ynETHWithdrawalQueueManager));
+            ynETHRedemptionAssetsVaultInstance.transferRedemptionAssets(_to, _amount);
+            assertEq(address(ynETHRedemptionAssetsVaultInstance).balance, _balanceBefore - _amount, "testTransferRedemptionAssets: E0");
+            assertEq(address(_to).balance - _toBalanceBefore, _amount, "testTransferRedemptionAssets: E1");
+        }
+    }
+
+    // function testWithdrawRedemptionAssets(uint256 _amount) public {
```

Comment:
end state of the PR cannot contain commented out code, unless it's a draft.

We won't be merging commented out code

We need to clarify if this is a valid test and it needs to be fixed up, or delete it

Reply:
code needs to be fixed in order for those to pass...

Reply:
Leave it as a failing test then to show case the test failure

Reply:
fixed in https://github.com/yieldnest/yieldnest-protocol/pull/124#discussion_r1671918664.